chatbot/python/chatbot.py

```diff
@@ -256,7 +256,7 @@ def client_message_loop(stream):
                 exec_future(msg.ctrl.id, msg.ctrl.code, msg.ctrl.text, msg.ctrl.params)
 
             elif msg.HasField("data"):
-                # log("message from:", msg.data.from_user_id)
+                # log("message from:", msg.data.from_user_id, "botUID:", botUID)
```

Comment:
I don't think this change is needed.

Reply:
Yes, it is just for debugging.

Reply:
Yes, I understand, but you don't need to change an existing line. Or you can remove it alltogether.

Reply:
Removed the change.